release-notes/opensearch-release-notes-2.0.0.md

```diff
@@ -0,0 +1,637 @@
+# OpenSearch and Dashboards 2.0.0 Release Notes
+
+## Release Highlights
+
+* Document level alerting allows users to create monitors that can generate alerts per document.
+* Lucene 9 is now used in OpenSearch
+* The Geo Map Tiles in OpenSearch Dashboards are updated and now have a pipeline to update them more frequently.
+* Document level security now supports term lookup queries.
+
+### OpenSearch Notifications
+* OpenSearch 2.0.0 is the first official release with OpenSearch Notifications.
+* Notifications consist of three plugins, `notifications-core` and `notifications` backend plugins for OpenSearch, and a `notificationsDashboards` frontend plugin for OpenSearch Dashboards.
+
+
+## Release Details
+
+OpenSearch and OpenSearch Dashboards 2.0.0 include the following features, enhancements, bug fixes, infrastructure, documentation, maintenance, and refactoring updates.
+
+OpenSearch [Release Notes](https://github.com/opensearch-project/OpenSearch/blob/2.0/release-notes/opensearch.release-notes-2.0.0.md).
+
+OpenSearch Dashboards [Release Notes](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/2.0/release-notes/opensearch-dashboards.release-notes-2.0.0.md).
+
+## FEATURES
+
+### OpenSearch Index Management
```

Comment:
Can we combine the UI line item below with this one?

Reply:
I think the UI PR was for the IM dashboards plugin so we probably want them separate.